woocommerce/woocommerce-android
# Home Screen: Add Switch Store option on More screen

Closes: #5583 


### Description
This adds the Switch Store option on top left (on LTR) of the More Screen.

The Switch Store option opens the Store Picker and allows merchants to change sites.

Behind the scene, it adds a view model as source of truth for the composable component's state.

Please ignore the styling of the text for now; the top left area will be updated with more contents soon.

<img src="https://user-images.githubusercontent.com/266376/151325465-6fa47996-a583-4f80-8e47-9a53cfaa2250.png" width="42%" />


### Testing instructions


1. Go to More screen,
2. Ensure "Switch Store" is shown on top left,
3. Click it, ensure Site Picker appears,
4. Switch to a different site, wait for verification and until it returns to the More Screen,
5. On More screen, tap WooCommerce Admin and View Store buttons. Ensure they both open to the right URLs (should use URL of the newly selected store),
6. Go to home screen and other tabs, ensure they now use and show information from the newly selected site.
